client/components/review-fields.js

```diff
@@ -42,6 +42,8 @@ const ReviewFields = ({ fields, values = {}, onEdit, editLink, project, prefix =
                 noComments={ noComments }
                 altLabels={altLabels}
                 readonly={readonly}
+                licenceHolder={licenceHolder}
+                establishment={establishment}
```

Comment:
This looks like it's going to put the licence holder information into every single field, which seems weird.

I'm wary of making these kind of changes to `<ReviewFields/>` and `<ReviewField/>` which are used in a lot of different places in order to deliver putting one piece of content onto one page.

I think adding the licence holder directly to the `legacy-introduction-review.js` file without going via these extra components would be a cleaner solution.

Reply:
@lennym - I agree, I struggled with it myself - the reason for passing properties to  ReviewFields is because we want to render the link to user profile below project title, and as it is not a field in schema , if I add the licence holder directly to the legacy-introduction-review.js, I have to render it either above or below all fields https://github.com/UKHomeOffice/asl-projects/blob/92f0f46dbd8e96a1e8204a3081a652edb1ed2006/client/pages/sections/legacy-introduction-review.js#L22, 
would *above* be more acceptable ?